drivers/sensor/lsm6dso/lsm6dso.c

```diff
@@ -791,9 +827,10 @@ static int lsm6dso_init(const struct device *dev)
 	}
 
 #ifdef CONFIG_LSM6DSO_SENSORHUB
+	data->shub_inited = true;
 	if (lsm6dso_shub_init(dev) < 0) {
-		LOG_DBG("failed to initialize external chip");
-		return -EIO;
+		LOG_INF("failed to initialize external chips");
```

Comment:
Not ERR? Seems a bit weird to log this at INF level.

Reply:
Sure. The only issue is that (and I think we discussed already about it) using this approach is easy, but I cannot discriminate whether the fail is because we are trying to init the sensorhub part for a LSM6DSO instance which is not or, instead, if it actually is a sensorhub but there is some error. To be honest I'm ready to accept this limitation, as an error like that is really rare. In the latter case a LOG_INF will be just returned.

Reply:
So, I will go changing "failed to initialize external chips" into "shub: no external chips found"